templates/deleteWebsite.twig

```diff
@@ -0,0 +1,10 @@
+<p>{{ 'TagManager_DeleteWebsiteExplanationLine1'|translate }}</p>
+<p>{{ 'TagManager_DeleteWebsiteExplanationLine2'|translate }}</p>
+<ol style="list-style-type:decimal !important;padding-left: 1rem;">
+    {% for container in containers %}
+        <li>
+            <a href="{{ link }}&idContainer={{ container.idcontainer }}" target="_blank" rel="noreferrer noopener">{{ container.name|e('html')|e('html') }}</a>
```

Comment:
@snake14 This was done on purpose to avoid xss being executed
Try creating a new container with name `Test <img src=x> Imm` and when you try to delete the corresponding site and if we escape only once the xss is executed and with double escaping it doesn't executes at all.

1. With single escape
![Screenshot from 2024-08-08 13-51-13](https://github.com/user-attachments/assets/2262e097-f2a0-47f2-9a92-c69b50d956b1)

2. With double escape
![Screenshot from 2024-08-08 13-48-51](https://github.com/user-attachments/assets/faec31e5-dcbe-4f70-86c6-d40f9033ca0c)

In 1st case the broken image gets displayed which is the probable xss

Reply:
Seems like an indication that maybe it should be a Vue template instead of Twig template, but it works :+1: